linode_api4/objects/serializable.py

```diff
@@ -54,28 +57,57 @@ class JSONObject(metaclass=JSONFilterableMetaclass):
         )
     """
 
+    always_include: ClassVar[Set[str]] = {}
+    """
+    A set of keys corresponding to fields that should always be
+    included in the generated output regardless of whether their values
+    are None.
+    """
```

Comment:
I was considering implementing something more similar to `omitempty` in Go, but in this case I think there are far fewer instances where we would want to include null values rather than excluding them.